index.js

```diff
@@ -109,6 +109,7 @@ var seenSnapshotNames = [];
 var buildResourceUploadPromises = [];
 var snapshotResourceUploadPromises = [];
 var isPercyEnabled = true;
+var createPercyBuildInvoked = false;
```

Comment:
Maybe `isOutputReady`? Or something that is more semantic and less about the explicit flow that was taken.

Reply:
I started with something like `isOutputReady`, and switched it to better represent what we actually care about (has the Percy build creation process started?) - which seemed to clearer express intent.  I didn't call it percyBuildCreated, as sometimes (i.e. no token) we eject and don't create the build.